resources/views/includes/slider/scripts.blade.php

```diff
@@ -31,6 +31,12 @@ function disableTabIndexOfInvisibleElements (sliderWrapper, slides) {
     }
 
     new Swiper('#swiper-{{ $id }}', {
+        @if($keyboardScroll)
+        keyboard: {
+            enabled: true,
+            onlyInViewport: true,
+        },
```

Comment:
@alexbarnsley By default, the keyboard navigation uses `pageUp` and `pageDown` keys to navigate (as you can see in the [documentation](https://swiperjs.com/swiper-api#keyboard-control)).

I think the purpose of the task is to optionally add the arrow (left/right) navigation.
Similar to this https://github.com/ArkEcosystem/marketsquare.io/blob/e325d3c6d90f39d5e078821076756a35cc83800e/resources/views/components/gallery/modal.blade.php#L8

Maybe you have to try with `pageUpDown: false`, idk if switch to arrow (left/right) keys 👇🏽 
```suggestion
        keyboard: {
            enabled: true,
            pageUpDown: false
        },
```

you can omit `onlyInViewport` because is `true` by default.

Reply:
thanks @leMaur!